### DISCUSSION TITLE
Where can i get the API key for Jan local server?

### DISCUSSION THREAD

**Discussion by ux-han** [janhq/jan#3612] [Get Help] [answered]
How do I obtain the API key for Jan's local server? Also, is it possible to access the server from other devices on my local network, or is it limited to the host PC?

**Comment by ux-han** [comment, marked as answer]
1. API key: For the local Jan server, you don't need a real API key. Just use any random text (like "sk-test") as the API key in GUI clients. Some OpenAI-compatible GUI clients require an API key, so this workaround should do the trick.

2. Network access: Currently, Jan's local server is designed to run on your PC only. We don't have built-in support for accessing it over your local network yet. But it's a cool idea! Maybe we could add that in a future update if enough people want it.

**Comment by StackPie71** [comment]
Hey, I think I've been trying to do this for the last 2 hours. I want to host my server on my tower and access it from my laptop when I'm not at home. I managed to connect to the API GUI using Tailscale, but it doesn't seem possible to tell my laptop's Jan's client to use this server. I would love to see this feature implemented.
Thanks for you work!

**Reply by brunoalbertopeixoto** [reply]
> Hey, I think I've been trying to do this for the last 2 hours. I want to host my server on my tower and access it from my laptop when I'm not at home. I managed to connect to the API GUI using Tailscale, but it doesn't seem possible to tell my laptop's Jan's client to use this server. I would love to see this feature implemented. Thanks for you work!

I would also like to do something like that. I'm developing an educational product: a chatbot that would work on Android, but would be powered by Jan's AI. Does anyone have any tutorials?

**Reply by louis-jan** [reply]
> Hey, I think I've been trying to do this for the last 2 hours. I want to host my server on my tower and access it from my laptop when I'm not at home. I managed to connect to the API GUI using Tailscale, but it doesn't seem possible to tell my laptop's Jan's client to use this server. I would love to see this feature implemented. Thanks for you work!

Hi @StackPie71, you can try using the OpenRouter extension. First, edit the chat/completions URL to point to the IP address of Jan Server. Then, input the model name that you are running on the server.
